Environment_Engine/Query/Space.cs

```diff
@@ -44,7 +44,30 @@ public static partial class Query
         [Output("panelsAsSpace", "A collection of Environment Panels which have the given space name as a connected space")]
         public static List<Panel> ToSpace(this List<Panel> panels, string spaceName)
         {
-            return panels.Where(x => x.ConnectedSpaces.Contains(spaceName)).ToList();
+            List<Panel> result= panels.Where(x => x.ConnectedSpaces.Contains(spaceName)).ToList();
+            if(result.Count==0)
+                BH.Engine.Reflection.Compute.RecordWarning("Could not find following space: " + spaceName);
+            if(result.Count > 0 && result.Count<5)
```

Comment:
Delete this, there is no issue raised for this use case and it does not aid anything to the user.